### ISSUE TITLE
Make compatible with ReverseDSC 1.9.4.4

### ISSUE THREAD

**Issue by heinrich-ulbricht** [microsoft/WebAdministrationDSC.Reverse#8]
ReverseDSC 1.9.4.4 currently seems not supported, the script throws one error over and over again:

_Get-DSCBlock : A parameter cannot be found that matches parameter name 'Indent'._

The parameters of `Get-DSCBlock` must have been changed in some version >1.9.3.0.

**Comment by Sibu08-2019** [comment]
I am planing for IIS website migration from 2008 to 2016. Is this script going to help me on this?

**Comment by NikCharlebois** [comment]
@Sibu08-2019 yes absolutely. ReverseDSC for IIS will allow you to extract the configuration of any environment and replicate it over another one. You could therefore extract from your 2008 server (assuming you have WMF 4+ installed) and then replicate over your 2016 server.

Now, this project will eventually become integrated natively as part of xWebAdministration via the following PR. I recommend you use that code base instead:
https://github.com/PowerShell/xWebAdministration/pull/442

**Comment by Sibu08-2019** [comment]
Hi Nik,

Looks like I am missing something. I have download and code and running locally on iis server. Script just ran and it didn't create mof file. Can you give me instruction to run locally first ?
